R/bootstrap.R

```diff
@@ -1545,10 +1555,16 @@ icon <- function(name, class = NULL, lib = "font-awesome") {
   iconTag <- tags$i(class = iconClass)
 
   # font-awesome needs an additional dependency (glyphicon is in bootstrap)
-  if (lib == "font-awesome") {
-    htmlDependencies(iconTag) <- htmlDependency(
-      "font-awesome", "4.7.0", c(href="shared/font-awesome"),
-      stylesheet = "css/font-awesome.min.css"
+  if (lib %in% c("font-awesome", "fontawesome")) {
```

Comment:
Hm I'm not sure about this. Perhaps we should do this:

- If it's `font-awesome`, load the shim
- If it's `fontawesome`, don't load the shim

Then in subsequent versions of Shiny that ship V5 w/out the shim, specifying `font-awesome` is an error and `fontawesome` is the new deault `lib`

Reply:
A half-baked scheme for supporting 4 and 5 in a subsequent release, but there's no reason to do it now. Removed and simplified.